gpt_index/tools/migrate_v1_to_v2.py

```diff
@@ -0,0 +1,213 @@
+"""Tool for migrating Index built with V1 data structs to V2."""
+import json
+from typing import Dict, List, Optional, Tuple, Type
+
+try:
+    import fire
+except ImportError:
+    print('Please run `pip install fire`')
```

Comment:
any particular reason for this?

Reply:
Want to use the Fire for CLI, but didn't want to add the dependency to `requirements.txt`